### DISCUSSION TITLE
posts/ota-tv-with-plex/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#275] [Comments]
# posts/ota-tv-with-plex/

Cut the cord and get free over the air TV with Plex! Today we’ll dive deep into selecting a TV tuner, an antenna, dialing in your TV signal, and configuring Plex to help you get the most out of Live TV and DVR!

https://boomsatyurij19.github.io/posts/ota-tv-with-plex/

**Comment by boomsatyurij19** [comment]
test comment its cool

**Reply by timothystewart6** [reply]
Test reply, thanks

**Comment by pmyshkin-git** [comment]
Thanks for the write-up!  Any recommendations on hardware for the Plex Server?  Is a Pi5 enough, or need some more power for transcoding, etc.

**Reply by timothystewart6** [reply]
Hey!  I am not sure if the Pi5 will be able to handle transcoding, let alone multiple streams.  TBD on real world perf.  Your best best is to either get an Intel CPU that supports QuickSync or use an NVIDIA card that support NVENC (most of them).  The video card can even be an older one, it just needs to support NVENC.
